lib/pdc/version.rb

```diff
@@ -1,3 +1,3 @@
 module PDC
-  VERSION = '0.1.6'.freeze
+  VERSION = '0.1.7'.freeze
```

Comment:
should be in a different commit to keep commits atomic as much as possible. 
Alternatively, lets remove bumping up the version since it is part of the release process.